src/frontend/magnify/src/components/index.ts

```diff
@@ -1 +1,2 @@
 export { default as TestButton, TestButtonVariant } from './TestButton';
+export * from "./sidebar"
```

Comment:
For the capital letter, it was a mistake I did: in the project we currently use kebab-case for components higher folders (like sidebar) and PascalCase for components themselves (like SidebarButton). In the latest push I changed the folder name.

Reply:
Concerning the `{ LayoutWithSidebar }` part, we thought with @Leonils that since it's a component bank meant to be used independently from the demo, it could be a good idea to export as most components as we can, and not only the ones we need in the demo. What do you think ?

Reply:
I'm ok to export `*` as long as it is done on purpose.